ostree-upload-server.py

```diff
@@ -85,7 +85,7 @@ def __init__(self, import_name, users, repo, upload_counter,
         self.route("/upload", methods=["GET", "POST"])(self.upload)
         self.route("/push", methods=["GET", "PUT"])(self.push)
 
-        self._tempdir = tempfile.mkdtemp(prefix="ostree-upload-server-")
+        self._tempdir = tempfile.mkdtemp(prefix="/var/tmp/ostree-upload-server-")
```

Comment:
I think you want `dir='/var/tmp'` rather than extending the prefix.

https://docs.python.org/2/library/tempfile.html#tempfile.mkstemp

Also, a comment about why this is in /var/tmp (potentially large uploads that would cause issues if /tmp is a tmpfs) would be nice.